dev.sh

```diff
@@ -1,4 +1,45 @@
 #!/bin/bash
 
-./db.sh
-pnpm run dev
\ No newline at end of file
+
+if [ "$EUID" -ne 0 ];
+  then echo "Please run as root. This is due to some overriding in certificates and the hosts file"
+  exit
+fi
+
+
+trap '' INT
+
+if [ ! -x "$(command -v mkcert)" ];
+then
+    curl -JLO "https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-linux-amd64"
+    chmod +x mkcert-v*-linux-amd64
+    cp mkcert-v*-linux-amd64 /usr/local/bin/mkcert
+    rm mkcert-v*-linux-amd64
+fi
+if [[ ! -f localhost.cert ]]
+then
+    mkcert -install
+    mkcert -cert-file localhost.cert -key-file localhost.key ni.fe.up.pt
+fi
+
+
+init_containers () (
+    trap - INT
+    if [ -x "$(command -v podman-compose)" ]; then
+        podman-compose up web-dev
+    else
+        docker-compose up web-dev
+    fi
```

Comment:
Shouldn't the user have the choice on which one to use? Also, on recent versions of docker, the command `docker-compose` does not exist and is instead replaced by `docker compose`.